### DISCUSSION TITLE
Zola port

### DISCUSSION THREAD

**Discussion by epogrebnyak** [thuliteio/doks#408] [Showcase]
Just noticed - https://www.getzola.org/themes/adidoks/ a port of Doks to Zola static site generator. References Doks, sign of popularity. 

With port I always wonder how do they keep up with updating with original theme, maybe it is easier with Doks.

**Comment by stevenroose** [comment]
The project is already abandoned it seems, so yeah updating with upstream seems to be a real problem.
